packages/front-end/components/HistoryTable.tsx

```diff
@@ -64,34 +64,64 @@ function EventDetails({
   return <Code language="json" code={JSON.stringify(json, null, 2)} />;
 }
 
-function HistoryTableRow({
+export function HistoryTableRow({
   event,
   open,
   setOpen,
+  isActivity = false,
+  experimentName = "",
 }: {
   event: AuditInterface;
   open: boolean;
   setOpen: (open: boolean) => void;
+  isActivity?: boolean;
+  experimentName?: string;
 }) {
+  let itemName = event.entity.id;
+  if (experimentName) {
+    itemName = experimentName;
+  }
   return (
     <>
       <tr
         onClick={() => {
-          if (event.details) setOpen(!open);
+          if (event.details && !isActivity) setOpen(!open);
         }}
         style={{ cursor: event.details ? "pointer" : "" }}
         className={open ? "highlight" : event.details ? "hover-highlight" : ""}
       >
         <td title={datetime(event.dateCreated)}>{ago(event.dateCreated)}</td>
+        {isActivity && (
+          <>
+            <td>{event.entity.object}</td>
+            <td>
+              <a
+                href={
+                  experimentName
+                    ? `/experiment/${event.entity.id}`
+                    : `/features/${event.entity.id}`
+                }
+              >
+                {" "}
+                {itemName}{" "}
```

Comment:
Get rid of these extra spaces.  If you need more padding, it should applied to the `<td>` element instead using the `px-1` class (or similar).